templates/react/src/example/Example.tsx

```diff
@@ -4,7 +4,7 @@ import { ethers } from 'ethers'
 import { AlphaRouter, ChainId, SwapType } from '@uniswap/smart-order-router'
 import { TradeType, CurrencyAmount, Percent } from '@uniswap/sdk-core'
 import { Environment, CurrentConfig } from '../config'
-import { getCurrencyBalance } from '../libs/wallet'
+import { getCurrencyBalance } from '../libs/balance'
```

Comment:
I was planning to add some other functions like wrapping/unwrapping ETH in some cases - can we keep this as wallet? The ABI rename makes sense still but we shouldn't keep this only to balance IMO

Reply:
Addressed below.